R/mod_safetyGraphicsServer.R

```diff
@@ -48,7 +49,28 @@ safetyGraphicsServer <- function(input, output, session, meta, mapping, domainDa
         )
     )
 
-    #Setting tab
+    # Keep chart status updated
+    charts_status <- reactive({
+        charts %>% purrr::map(function(chart){
+            if(hasName(chart, 'dataSpec')){
```

Comment:
@jwildfire Will this piece be updated to check status on charts w/o the dataSpec input as they are configured manually on mapping tab?

Reply:
My plan was just to ignore chart objects where `dataSpec` isn't defined. The behavior for those charts would be the same as in v2.1. Basically, they'd try to re-render whenever mapping updates, and if they bomb they bomb ... 

I'm open to suggestions though!

Reply:
@jwildfire Makes sense! I'm sure you explained that to me, but as a user I was expecting them to show. Could it be a future enhancement?

Reply:
I think my problem is that I don't have any good ideas about how to check status if no `dataSpec` is included in the chart object ... We could have an icon or note in the UI indicating that no checking is done for some charts, but not sure what to do beyond that.